java/ingest/ingest-core/src/main/java/sleeper/ingest/job/IngestJobValidationUtils.java

```diff
@@ -37,16 +42,20 @@ public static <T> Optional<T> deserialiseAndValidate(
         T job;
         try {
             job = deserialiser.apply(message);
+            LOGGER.info("Job deserialised successfully");
```

Comment:
This duplicates a similar log message in IngestJobQueueConsumer, though it seems to make some sense to log it here instead. Maybe we should remove the other one, and make this log out the whole job object?